src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx

```diff
@@ -49,14 +49,14 @@ const InformationLeftColumn: React.FunctionComponent<PropsType> = ({
               isLeftColumnExpanded ? "expanded-popup" : ""
             }`}
           />
-        ) : venueLogoPath === "heart" && isLeftColumnExpanded ? (
-          <img
-            src="/imageDonate1.png"
-            alt="experience-logo"
-            className={`expanded-popup`}
+        ) : venueLogoPath === "heart" ? (
+          <FontAwesomeIcon
```

Comment:
Would like to understand a little more where the heart went 😂 considering this component is used in many places in the code.

Reply:
> Would like to understand a little more where the heart went 😂 considering this component is used in many places in the code.

This version is putting the heart back, I removed the burning man project graphic that appeared there before.

Reply:
Aha thanks!